README.md

```diff
@@ -33,46 +33,18 @@ public class MyClass {
 
 ### Including 3rd Party Libraries
 
```

Comment:
It would be better to mention: If you are generating your project using the latest Azure functions java archetype, the dependencies section is automatically created for you.